dist/index.html

```diff
@@ -6,35 +6,43 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Muhammad Salyhu</title>
     <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
 </head>
 <body>
```

Comment:
- [ ] [OPTIONAL] I advise you to make the HTML scroll behavior smooth, this will make it look more pleasing and feel more enticing 😀.

```
html {
  scroll-behavior: smooth;
}
```